LibTrixi.jl/src/api_jl.jl

```diff
@@ -1,4 +1,4 @@
-function trixi_initialize_jl(filename)
+function trixi_setup_simulation_jl(filename)
```

Comment:
It's a little 🚲🛖 (bike shedding), but "setup" imho is a noun, not a verb. Thus it should be either something like
```suggestion
function trixi_create_simulation_jl(filename)
```
or at least
```suggestion
function trixi_set_up_simulation_jl(filename)
```
If you ask me, the latter one has too many underscores, thus I'd personally prefer the first version (or a variant thereof).

Note that if this is changed here, it should be changed consistently for all similarly-named functions